packages/fxa-settings/src/components/PageDisplayName/index.tsx

```diff
@@ -3,37 +3,60 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { RouteComponentProps } from '@reach/router';
-import React, { useRef } from 'react';
+import { useAccount } from 'fxa-settings/src/models';
+import React, { ChangeEvent, useRef, useCallback, useState } from 'react';
 import FlowContainer from '../FlowContainer';
 import InputText from '../InputText';
 
+const validateDisplayName = (currentDisplayName: string) => (
+  newDisplayName: string
+) => newDisplayName !== '' && newDisplayName !== currentDisplayName;
+
 export const PageDisplayName = ({}: RouteComponentProps) => {
   const displayNameRef = useRef<HTMLInputElement>(null);
+  const user = useAccount();
+  const isValidDisplayName = validateDisplayName(user.displayName || '');
+  const [saveBtnDisabled, setSaveBtnDisabled] = useState(true);
+
+  const checkForm = useCallback(
+    (ev: ChangeEvent<HTMLInputElement>) => {
+      setSaveBtnDisabled(!isValidDisplayName(ev.target.value));
+    },
+    [setSaveBtnDisabled, isValidDisplayName]
+  );
```

Comment:
This is fine. Just fyi I added [react-hook-form](https://react-hook-form.com/) while doing the more complicated validation for change password. It might make sense to use it throughout settings, but it doesn't need to be now.

Reply:
I can't seem to find where react-hook-form is currently used in the repo.  😕

Reply:
https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/PageChangePassword/index.tsx#L55